idaes_examples/notebooks/docs/diagnostics/structural_singularity.ipynb

```diff
@@ -0,0 +1,767 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "a261df3e-3957-4a85-ac6d-1bbe9eb053f8",
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "ModuleNotFoundError",
```

Comment:
The first cell here seems to have some error regarding the module not found.

Reply:
Okay, I was testing something and must have committed that by mistake. I'll clear this when I'm back on a computer with Jupyter.